### PR TITLE
DCOS: added tags to VMs in cluster

### PR DESCRIPTION
**Pull request by dmitsh**
DCOS: added tags to VMs in cluster

### CODE REVIEW

File: parts/dcos/dcosagentresourcesvmss.t [hunk 1]
```diff
@@ -102,10 +102,13 @@
       ],
       "tags":
       {
-        "creationSource" : "[concat('acsengine-', variables('{{.Name}}VMNamePrefix'), '-vmss')]"
+        "creationSource" : "[concat('acsengine-', variables('{{.Name}}VMNamePrefix'), 'vmss')]",
```
Comment (radhikaj) [inline parts/dcos/dcosagentresourcesvmss.t]: Can we add some tag to say this is a Windows agent. Thanks.

File: parts/dcos/dcosmastervars.t [hunk 2]
```diff
@@ -26,6 +26,7 @@
         {{end}}
       ],
 {{end}}
+    "orchestratorVersion": "{{.OrchestratorProfile.OrchestratorVersion}}",
```
Comment (radhikaj) [inline parts/dcos/dcosmastervars.t]: Here in agentWindowsVersion can we tag with whether it is a Windows or a Linux agent?

Comment (dmitsh) [reply parts/dcos/dcosmastervars.t]: OS type has its own field

**Pull request merged**